oaknational/Oak-Web-Application
# [PUPIL-793] Lesson review - update buttons

## Description

Music year: 1958

This swaps the buttons on the lesson review page.

[Ticket here](https://www.notion.so/oaknationalacademy/Lesson-review-update-buttons-and-links-Beano-findings-73e928b7f7814467a3a3218feda915a8)

## How to test

1. Go to https://deploy-preview-2622--oak-web-application.netlify.thenational.academy/pupils/programmes/english-primary-year-4/units/simple-compound-adverbial-and-relative-complex-sentences/lessons/three-sentence-types/review
2. There should be a link in the top left "Lesson overview" that takes you to the lesson overview
3. There should be a link in the bottom right "View all lessons" that takes you to all the lessons in that unit


## Checklist
